Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/SwaggerVer2.cs

```diff
@@ -36,6 +36,8 @@ public class SwaggerVer2 : OpenApiBase
         public ObservableList<ApplicationAPIModel> SwaggerTwo(SwaggerDocument Swaggerdoc, ObservableList<ApplicationAPIModel> SwaggerModels)
         {
             swagTwo = Swaggerdoc;
+            var listofExamples = GetExamplesFromDefinitions(swagTwo);
```

Comment:
Please use same comments from OpenApiVer3 class to SwaggerVer2  class

Reply:
done.